gcp/live/dev/k8s/gpii/couchdb-prometheus-exporter/terraform.tfvars

```diff
@@ -0,0 +1,24 @@
+# ↓ Module metadata
+terragrunt = {
+  terraform {
+    source = "/project/modules//couchdb-prometheus-exporter"
+  }
+
+  dependencies {
+    paths = [
+      "../couchdb",
+    ]
+  }
+
+  include = {
+    path = "${find_in_parent_folders()}"
+  }
+}
+
+# ↓ Module configuration (empty means all default)
+
+replica_count     = 1
+requests_cpu      = "25m"
+requests_memory   = "128Mi"
+limits_cpu        = "50m"
```

Comment:
I remember this conversation from a different PR, but I thought we must have settled on the current scheme (memory the same, allow cpu to burst) because I copied these values from gpii-preferences.

I added a Parking Lot item to discuss this more broadly. Then, I'm happy to fix this everywhere in a follow-up PR (I noticed another small thing in gpii-preferences that I'd like to fix anyway).

Reply:
I think we had a problem to figure the "working" values out. I still believe having same "requests" and "limits" for these types of workloads is a way to get more predictable behavior, but happy to discuss at standup.